data/pom.xml

```diff
@@ -98,7 +98,12 @@
             <version>${mockito.version}</version>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
```

Comment:
Can't find where you're using this...

Reply:
https://github.com/Azure/azure-kusto-java/pull/57/files#diff-dfdcac39245e8c607ad8d656e094db82R32